modules/nw-cno-view.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * networking/cluster-network-operator.adoc
+
+[id="nw-cluster-network-view_{context}"]
+= View the cluster network configuration
+
+Every new {product-title} installation has a `network.operator` named `cluster`.
+
+.Procedure
+
+* Use the `oc describe` command to view the cluster network configuration:
++
+----
+oc describe network.operator/cluster
+Name:         cluster
+Namespace:
+Labels:       <none>
+Annotations:  <none>
+API Version:  operator.openshift.io/v1
+Kind:         Network
+Metadata:
+  Self Link:           /apis/operator.openshift.io/v1/networks/cluster
+Spec:
+  Cluster Network:
+    Cidr:         10.128.0.0/14
+    Host Prefix:  23
+  Default Network:
+    Type:  OpenShiftSDN
+  Service Network:
+    172.30.0.0/16
+Status:
```

Comment:
It would be good to discuss the status field as well.

Reply:
@squeed @danwinship, what can I say about the `Status` field?

I'd like to have this ready by EOB on Friday for the GA next week. Any information you can provide is appreciated. Thanks!